lib/core/src/Cardano/Pool/Ranking.hs

```diff
@@ -104,11 +106,46 @@ desirability
     -> Double
 desirability constants pool
     | f_saturated <= c = 0
-    | otherwise    = (f_saturated - c) * (1 - m)
+    | otherwise        = (f_saturated - c) * (1 - m)
   where
     f_saturated = saturatedPoolRewards constants pool
     m = getRatio $ margin pool
-    c = fromIntegral $ getLovelace $ cost pool
+    c = fromIntegral $ getQuantity $ cost pool
+
+-- | The saturation-level of a pool indicate how far a pool is from the
+-- desirated relative stake fixed by the network. A saturation level above 1
+-- means that the pool is satured. A level of 0.5 / 50% means that the pool owns
+-- half the ideal stake.
```

Comment:
> half the ideal stake

Little suspicious of the wording here. Is `z0` to be viewed as the "ideal" stake?

Reply:
Yes. Ideal / Desired.